### PR TITLE
CONTRIBUTING: add code comment style

### PR DESCRIPTION
**Pull request by CaitinChen**
This PR will link to the comment style in the tikv repo.
It can only be merged when [tikv/tikv#4204](https://github.com/tikv/tikv/pull/4204) (waiting for Brian's review tomorrow) is merged.

### CODE REVIEW

**Comment by CaitinChen** [comment]
@lilin90 PTAL

**Comment by CaitinChen** [comment]
@lilin90 PTAL

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -256,6 +256,54 @@ review.
...
+Follow this style to write TiDB code comment.
+
+For the code comment style of the TiKV project, see [Code Comment Style](https://github.com/tikv/tikv/blob/master/CODE_COMMENT_STYLE.md).
+
+**Format:**
+
+- Use `//` for a single-line comment and trailing comment.
+- Use `/* ... */` for a block comment (used only when needed).
+- Use **gofmt** to format your code.
+- Place the single-line and block comment above the code it’s annotating.
```
Comment (lilin90) [inline CONTRIBUTING.md]:
```suggestion
- Place the single-line or block comment above the code it's annotating.
```

**Comment by CaitinChen** [comment]
@shenli PTAL

**Pull request merged**